src/nspawn/nspawn.c

```diff
@@ -378,6 +379,9 @@ static int help(void) {
                "%3$sSecurity:%4$s\n"
                "     --capability=CAP       In addition to the default, retain specified\n"
                "                            capability\n"
+               "     --ambient-capability=CAP\n"
+               "                            Sets the specified capability for the started\n"
+               "                            process. Not useful if booting a machine.\n"
                "     --drop-capability=CAP  Drop the specified capability from the default set\n"
```

Comment:
sorry for not noticing this earlier, but here too, please swap the latter two options, i.e. list `--capability=`/`--drop-capabaility=` first, and `--amvient-capability=` after that, since the first two operate on the same set of caps, and the other on another